great_expectations/expectations/registry.py

```diff
@@ -16,6 +16,10 @@
 
 import great_expectations.exceptions as ge_exceptions
 from great_expectations.core.id_dict import IDDict
+from great_expectations.core.metric_function_types import (
+    MetricFunctionTypes,
+    MetricPartialFunctionTypes,
+)
```

Comment:
These should be part of `if TYPE_CHECKING` imports

Reply:
@Kilo59 Good catch -- moved (in next commit).